tensorflow/python/ops/rnn.py

```diff
@@ -427,7 +427,7 @@ def _reverse(input_, seq_lengths, seq_dim, batch_dim):
       if seq_lengths is not None:
         return array_ops.reverse_sequence(
             input=input_, seq_lengths=seq_lengths,
-            seq_dim=seq_dim, batch_dim=batch_dim)
+            seq_axis=seq_dim, batch_axis=batch_dim)
```

Comment:
Also please rename variables from seq_dim to seq_axis, batch_dim to batch_axis accordingly. Thanks.

Reply:
Done!